.ci/build.sh

```diff
@@ -1,11 +1,48 @@
 #!/bin/bash
 set -e
 
+# set to one of the following values
+# - don't use HelmQA: (empty)
+# - use HelmQA in SaaS mode: http://helmqa-zhaw-prod-demos.appuioapp.ch/livecheck?repo
+# - use HelmQA locally in Docker mode: http://127.0.0.1:5000/livecheck?repo
+HELMQA_URL=http://helmqa-zhaw-prod-demos.appuioapp.ch/livecheck?repo
```

Comment:
Quote URLs in single-quotes (`'`) to avoid globbing

Reply:
done